app/services/assign_news_item_default_news_categories.rb

```diff
@@ -0,0 +1,27 @@
+class AssignNewsItemDefaultNewsCategories
+  def initialize(news_item:)
+    @news_item = news_item
+  end
+
+  def call
+    if is_coin_project_tweet?
+      project_announcements_category = NewsCategory.find_project_announcements!
+
+      # Assign if does not already exist
+      unless @news_item.news_categories.exists?(project_announcements_category.id)
+        @news_item.news_categories << project_announcements_category
+        @news_item.last_machine_tagged_on = Time.now
```

Comment:
@hanchang I just realized, this field is used for tagging with coin mentions and not categories right?

Reply:
@jessehon Yes, great catch - machine tagging only handles tagging Coins, there's no capability for machine tagging NewsCategories at the moment.

Reply:
So we should not mark NewsItems as `last_machine_tagged_on` since it's assumed that all "Project Announcements" NewsCategories will be automatically marked.

Reply:
Instead, the `last_machine_tagged_on` field should only be set by the NewsTagger application.